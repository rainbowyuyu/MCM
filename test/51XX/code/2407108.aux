\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\abx@aux@refcontext{apa/global//global/global}
\@writefile{toc}{\contentsline {section}{\numberline {1}Introduction}{2}{section.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {1.1}Background}{2}{subsection.1.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {1.2}Restatement of the Problem}{2}{subsection.1.2}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {1.3}Our Work}{3}{subsection.1.3}\protected@file@percent }
\@writefile{lof}{\contentsline {figure}{\numberline {1}{\ignorespaces Work Process\relax }}{3}{figure.caption.1}\protected@file@percent }
\providecommand*\caption@xref[2]{\@setref\relax\@undefined{#1}}
\newlabel{fig1}{{1}{3}{Work Process\relax }{figure.caption.1}{}}
\@writefile{toc}{\contentsline {section}{\numberline {2}Assumptions and Justification}{3}{section.2}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {3}Notations}{3}{section.3}\protected@file@percent }
\@writefile{lot}{\contentsline {table}{\numberline {1}{\ignorespaces Notations\relax }}{4}{table.caption.2}\protected@file@percent }
\newlabel{tab1}{{1}{4}{Notations\relax }{table.caption.2}{}}
\@writefile{toc}{\contentsline {section}{\numberline {4}Data Preprocessing}{4}{section.4}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {4.1}Data Cleaning}{4}{subsection.4.1}\protected@file@percent }
\@writefile{lof}{\contentsline {figure}{\numberline {2}{\ignorespaces $SpeedNA_Count_By_RallyCount$\relax }}{5}{figure.caption.3}\protected@file@percent }
\newlabel{fig2}{{2}{5}{$SpeedNA_Count_By_RallyCount$\relax }{figure.caption.3}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {4.2}Data Cleaning}{5}{subsection.4.2}\protected@file@percent }
\@writefile{lof}{\contentsline {figure}{\numberline {3}{\ignorespaces Boxplot of Serve Speed By Serve Number\relax }}{6}{figure.caption.4}\protected@file@percent }
\newlabel{fig3}{{3}{6}{Boxplot of Serve Speed By Serve Number\relax }{figure.caption.4}{}}
\@writefile{toc}{\contentsline {section}{\numberline {5}Model Construction}{6}{section.5}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {5.1}Discrete Performance Evaluating Model Based on Random Forest}{6}{subsection.5.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsubsection}{\numberline {5.1.1}Model Preparation}{6}{subsubsection.5.1.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsubsection}{\numberline {5.1.2}Random Forest Model}{7}{subsubsection.5.1.2}\protected@file@percent }
\@writefile{lof}{\contentsline {figure}{\numberline {4}{\ignorespaces Concept of Random Forest\relax }}{7}{figure.caption.5}\protected@file@percent }
\newlabel{fig4}{{4}{7}{Concept of Random Forest\relax }{figure.caption.5}{}}
\@writefile{lot}{\contentsline {table}{\numberline {2}{\ignorespaces Feature Importance Ranking in Random Forest Model\relax }}{8}{table.caption.6}\protected@file@percent }
\newlabel{tab2}{{2}{8}{Feature Importance Ranking in Random Forest Model\relax }{table.caption.6}{}}
\@writefile{lof}{\contentsline {figure}{\numberline {5}{\ignorespaces Feature Importance in Random Forest Model\relax }}{8}{figure.caption.7}\protected@file@percent }
\newlabel{fig5}{{5}{8}{Feature Importance in Random Forest Model\relax }{figure.caption.7}{}}
\newlabel{eq1}{{1}{8}{Random Forest Model}{equation.5.1}{}}
\newlabel{eq2}{{2}{8}{Random Forest Model}{equation.5.2}{}}
\newlabel{fig:first}{{6a}{9}{First Image\relax }{figure.caption.8}{}}
\newlabel{sub@fig:first}{{a}{9}{First Image\relax }{figure.caption.8}{}}
\newlabel{fig:second}{{6b}{9}{Second Image\relax }{figure.caption.8}{}}
\newlabel{sub@fig:second}{{b}{9}{Second Image\relax }{figure.caption.8}{}}
\@writefile{lof}{\contentsline {figure}{\numberline {6}{\ignorespaces Player 1/2’s Momentum in this set\_no\relax }}{9}{figure.caption.8}\protected@file@percent }
\newlabel{fig:twoimages}{{6}{9}{Player 1/2’s Momentum in this set\_no\relax }{figure.caption.8}{}}
\@writefile{lof}{\contentsline {figure}{\numberline {7}{\ignorespaces Player 1’s vs Player 2’s Momentum Comparison in the set\_no\relax }}{9}{figure.caption.9}\protected@file@percent }
\newlabel{fig6}{{7}{9}{Player 1’s vs Player 2’s Momentum Comparison in the set\_no\relax }{figure.caption.9}{}}
\@writefile{lof}{\contentsline {figure}{\numberline {8}{\ignorespaces : Player 1-Player’s Momentum Difference Comparison in this set\_no\relax }}{10}{figure.caption.10}\protected@file@percent }
\newlabel{fig7}{{8}{10}{: Player 1-Player’s Momentum Difference Comparison in this set\_no\relax }{figure.caption.10}{}}
\newlabel{fig:first}{{\caption@xref {fig:first}{ on input line 398}}{10}{Random Forest Model}{figure.caption.11}{}}
\newlabel{sub@fig:first}{{}{10}{Random Forest Model}{figure.caption.11}{}}
\newlabel{fig:second}{{\caption@xref {fig:second}{ on input line 403}}{10}{Random Forest Model}{figure.caption.11}{}}
\newlabel{sub@fig:second}{{}{10}{Random Forest Model}{figure.caption.11}{}}
\newlabel{fig:third}{{\caption@xref {fig:third}{ on input line 407}}{10}{Random Forest Model}{figure.caption.11}{}}
\newlabel{sub@fig:third}{{}{10}{Random Forest Model}{figure.caption.11}{}}
\newlabel{fig:forth}{{\caption@xref {fig:forth}{ on input line 411}}{10}{Random Forest Model}{figure.caption.11}{}}
\newlabel{sub@fig:forth}{{}{10}{Random Forest Model}{figure.caption.11}{}}
\@writefile{lof}{\contentsline {figure}{\numberline {9}{\ignorespaces Player 1/2’s Momentum in this set\_no\relax }}{10}{figure.caption.11}\protected@file@percent }
\newlabel{fig:twoimages}{{9}{10}{Player 1/2’s Momentum in this set\_no\relax }{figure.caption.11}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {5.2}Assessment Based on Logistic Regression}{10}{subsection.5.2}\protected@file@percent }
\@writefile{toc}{\contentsline {subsubsection}{\numberline {5.2.1}Assessment Preparation}{10}{subsubsection.5.2.1}\protected@file@percent }
\@writefile{lof}{\contentsline {figure}{\numberline {10}{\ignorespaces Player 1’s Momentum in this set\_no\relax }}{11}{figure.caption.12}\protected@file@percent }
\newlabel{fig7}{{10}{11}{Player 1’s Momentum in this set\_no\relax }{figure.caption.12}{}}
\newlabel{eq3}{{3}{11}{Assessment Preparation}{equation.5.3}{}}
\newlabel{eq4}{{4}{11}{Assessment Preparation}{equation.5.4}{}}
\newlabel{eq5}{{5}{11}{Assessment Preparation}{equation.5.5}{}}
\newlabel{fig:first}{{11a}{12}{Original Data Scatter Plot\relax }{figure.caption.13}{}}
\newlabel{sub@fig:first}{{a}{12}{Original Data Scatter Plot\relax }{figure.caption.13}{}}
\newlabel{fig:second}{{11b}{12}{Standardized Data Scatter Plot\relax }{figure.caption.13}{}}
\newlabel{sub@fig:second}{{b}{12}{Standardized Data Scatter Plot\relax }{figure.caption.13}{}}
\@writefile{lof}{\contentsline {figure}{\numberline {11}{\ignorespaces Two Types of Data Scatter\relax }}{12}{figure.caption.13}\protected@file@percent }
\newlabel{fig:twoimages}{{11}{12}{Two Types of Data Scatter\relax }{figure.caption.13}{}}
\newlabel{eq6}{{6}{12}{Assessment Preparation}{equation.5.6}{}}
\newlabel{LastPage}{{}{12}{}{page.12}{}}
\xdef\lastpage@lastpage{12}
\xdef\lastpage@lastpageHy{12}
\abx@aux@read@bbl@mdfivesum{5D0D039BF84E4F45A7144D280F172F5D}
\gdef \@abspage@last{14}
