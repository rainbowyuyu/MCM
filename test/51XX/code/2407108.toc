\contentsline {section}{\numberline {1}Introduction}{2}{section.1}%
\contentsline {subsection}{\numberline {1.1}Background}{2}{subsection.1.1}%
\contentsline {subsection}{\numberline {1.2}Restatement of the Problem}{2}{subsection.1.2}%
\contentsline {subsection}{\numberline {1.3}Our Work}{3}{subsection.1.3}%
\contentsline {section}{\numberline {2}Assumptions and Justification}{3}{section.2}%
\contentsline {section}{\numberline {3}Notations}{3}{section.3}%
\contentsline {section}{\numberline {4}Data Preprocessing}{4}{section.4}%
\contentsline {subsection}{\numberline {4.1}Data Cleaning}{4}{subsection.4.1}%
\contentsline {subsection}{\numberline {4.2}Data Cleaning}{5}{subsection.4.2}%
\contentsline {section}{\numberline {5}Model Construction}{6}{section.5}%
\contentsline {subsection}{\numberline {5.1}Discrete Performance Evaluating Model Based on Random Forest}{6}{subsection.5.1}%
\contentsline {subsubsection}{\numberline {5.1.1}Model Preparation}{6}{subsubsection.5.1.1}%
\contentsline {subsubsection}{\numberline {5.1.2}Random Forest Model}{7}{subsubsection.5.1.2}%
\contentsline {subsection}{\numberline {5.2}Assessment Based on Logistic Regression}{10}{subsection.5.2}%
\contentsline {subsubsection}{\numberline {5.2.1}Assessment Preparation}{10}{subsubsection.5.2.1}%
