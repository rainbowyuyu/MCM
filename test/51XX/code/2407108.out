\BOOKMARK [1][-]{section.1}{\376\377\000I\000n\000t\000r\000o\000d\000u\000c\000t\000i\000o\000n}{}% 1
\BOOKMARK [2][-]{subsection.1.1}{\376\377\000B\000a\000c\000k\000g\000r\000o\000u\000n\000d}{section.1}% 2
\BOOKMARK [2][-]{subsection.1.2}{\376\377\000R\000e\000s\000t\000a\000t\000e\000m\000e\000n\000t\000\040\000o\000f\000\040\000t\000h\000e\000\040\000P\000r\000o\000b\000l\000e\000m}{section.1}% 3
\BOOKMARK [2][-]{subsection.1.3}{\376\377\000O\000u\000r\000\040\000W\000o\000r\000k}{section.1}% 4
\BOOKMARK [1][-]{section.2}{\376\377\000A\000s\000s\000u\000m\000p\000t\000i\000o\000n\000s\000\040\000a\000n\000d\000\040\000J\000u\000s\000t\000i\000f\000i\000c\000a\000t\000i\000o\000n}{}% 5
\BOOKMARK [1][-]{section.3}{\376\377\000N\000o\000t\000a\000t\000i\000o\000n\000s}{}% 6
\BOOKMARK [1][-]{section.4}{\376\377\000D\000a\000t\000a\000\040\000P\000r\000e\000p\000r\000o\000c\000e\000s\000s\000i\000n\000g}{}% 7
\BOOKMARK [2][-]{subsection.4.1}{\376\377\000D\000a\000t\000a\000\040\000C\000l\000e\000a\000n\000i\000n\000g}{section.4}% 8
\BOOKMARK [2][-]{subsection.4.2}{\376\377\000D\000a\000t\000a\000\040\000C\000l\000e\000a\000n\000i\000n\000g}{section.4}% 9
\BOOKMARK [1][-]{section.5}{\376\377\000M\000o\000d\000e\000l\000\040\000C\000o\000n\000s\000t\000r\000u\000c\000t\000i\000o\000n}{}% 10
\BOOKMARK [2][-]{subsection.5.1}{\376\377\000D\000i\000s\000c\000r\000e\000t\000e\000\040\000P\000e\000r\000f\000o\000r\000m\000a\000n\000c\000e\000\040\000E\000v\000a\000l\000u\000a\000t\000i\000n\000g\000\040\000M\000o\000d\000e\000l\000\040\000B\000a\000s\000e\000d\000\040\000o\000n\000\040\000R\000a\000n\000d\000o\000m\000\040\000F\000o\000r\000e\000s\000t}{section.5}% 11
\BOOKMARK [3][-]{subsubsection.5.1.1}{\376\377\000M\000o\000d\000e\000l\000\040\000P\000r\000e\000p\000a\000r\000a\000t\000i\000o\000n}{subsection.5.1}% 12
\BOOKMARK [3][-]{subsubsection.5.1.2}{\376\377\000R\000a\000n\000d\000o\000m\000\040\000F\000o\000r\000e\000s\000t\000\040\000M\000o\000d\000e\000l}{subsection.5.1}% 13
\BOOKMARK [2][-]{subsection.5.2}{\376\377\000A\000s\000s\000e\000s\000s\000m\000e\000n\000t\000\040\000B\000a\000s\000e\000d\000\040\000o\000n\000\040\000L\000o\000g\000i\000s\000t\000i\000c\000\040\000R\000e\000g\000r\000e\000s\000s\000i\000o\000n}{section.5}% 14
\BOOKMARK [3][-]{subsubsection.5.2.1}{\376\377\000A\000s\000s\000e\000s\000s\000m\000e\000n\000t\000\040\000P\000r\000e\000p\000a\000r\000a\000t\000i\000o\000n}{subsection.5.2}% 15
